pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -113,6 +113,15 @@ type PrebindPlugin interface {
 	Prebind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
 }
 
+// PostbindPlugin is an interface that must be implemented by "postbind" plugins.
+// These plugins are called after a pod was scheduled with a cycle.
+type PostbindPlugin interface {
+	Plugin
+	// Postbind is called after binding a pod. All postbind plugins must return
+	// success, otherwise state associated with the reserved pod will not be cleaned up
+	Postbind(pc *PluginContext, p *v1.Pod, nodeName string) *Status
```

Comment:
Please remove `*Status`. They don't need to return anything. Postbind plugins are informational only. The scheduler does not process their return values.